lib/widgets/routers/settings_flow.dart

```diff
@@ -1,16 +1,16 @@
-import 'package:coffeecard/base/strings.dart';
 import 'package:coffeecard/widgets/pages/settings/settings_page.dart';
-import 'package:coffeecard/widgets/pages/settings/your_profile_page.dart';
 import 'package:flutter/material.dart';
 
 class SettingsFlow extends StatelessWidget {
   static Route get route => MaterialPageRoute(builder: (_) => SettingsFlow());
 
-  static const settingsRoute = '/';
-  static const yourProfileRoute = '/your-profile';
-
   static final navigatorKey = GlobalKey<NavigatorState>();
-  static void push(String route) => navigatorKey.currentState!.pushNamed(route);
+
+  static void push(Widget page) {
```

Comment:
I generally like this change to SettingsFlow, but maybe we should stop to consider the purpose of it? I never quite understood the reasoning for the old version of this, and I presume @fremartini didn't do so either, since I think he was the one who originally used the navigator to push to the stack, instead of the flow classes. 

While the new version is more straight forward, and easier to use, I don't see how it differs from the page_pusher that I introduced as a convenience method. If it is only supposed to be a convenience method, why are we even using multiple different ones? And if that is not, its only purpose, what else do you see as its purpose?

Reply:
OK i'm falling down the rabbit hole trying to come up with an elegant alternative, so let's discuss this in person